tools/benchmark/pnpm-lock.yaml

```diff
@@ -4,80 +4,78 @@ settings:
   autoInstallPeers: true
   excludeLinksFromLockfile: false
 
-importers:
-
-  .:
-    dependencies:
-      chai:
-        specifier: ^4.2.0
-        version: 4.3.7
-      chalk:
-        specifier: ^4.1.2
-        version: 4.1.2
-      easy-table:
-        specifier: ^1.1.1
-        version: 1.2.0
-      mocha:
-        specifier: ^10.0.0
-        version: 10.2.0
-      mocha-json-output-reporter:
-        specifier: ^2.0.1
-        version: 2.1.0(mocha@10.2.0)(moment@2.29.4)
-      mocha-multi-reporters:
-        specifier: ^1.5.1
-        version: 1.5.1(mocha@10.2.0)
-      moment:
-        specifier: ^2.21.0
-        version: 2.29.4
-    devDependencies:
-      '@fluid-internal/mocha-test-setup':
-        specifier: ~2.0.0-rc.3.0.3
-        version: 2.0.0-rc.3.0.3
-      '@fluidframework/build-common':
-        specifier: ^2.0.3
-        version: 2.0.3
-      '@fluidframework/eslint-config-fluid':
-        specifier: ^5.2.0
-        version: 5.3.0(eslint@8.55.0)(typescript@5.1.6)
-      '@microsoft/api-extractor':
-        specifier: ^7.45.1
-        version: 7.45.1(@types/node@14.18.38)
-      '@types/benchmark':
-        specifier: ^2.1.0
-        version: 2.1.2
-      '@types/chai':
-        specifier: ^4.3.4
-        version: 4.3.11
-      '@types/easy-table':
-        specifier: ^0.0.32
-        version: 0.0.32
-      '@types/mocha':
-        specifier: ^9.1.1
-        version: 9.1.1
-      '@types/node':
-        specifier: ^14.18.0
-        version: 14.18.38
-      concurrently:
-        specifier: ^8.2.1
-        version: 8.2.1
-      copyfiles:
-        specifier: ^2.4.1
-        version: 2.4.1
-      cross-env:
-        specifier: ^7.0.2
-        version: 7.0.3
-      eslint:
-        specifier: ~8.55.0
-        version: 8.55.0
-      prettier:
-        specifier: ~3.0.3
-        version: 3.0.3
-      rimraf:
-        specifier: ^2.6.2
-        version: 2.7.1
-      typescript:
-        specifier: ~5.1.6
-        version: 5.1.6
+dependencies:
```

Comment:
This PR doesn't have any dependency changes, so there shouldn't be any changes here. You'll want to revert this.